token/driver/request.go

```diff
@@ -35,25 +37,48 @@ func (r *TokenRequest) FromBytes(raw []byte) error {
 	return err
 }
 
+// IssueMetadata contains the metadata of an issue action.
+// In more details, there is an issuer and a list of outputs.
+// For each output, there is a token info and a list of receivers with their audit info to recover their enrollment ID.
```

Comment:
but this is true for anyone who receives the metadata, not just the auditor. Right @KElkhiyaoui?

Reply:
Since a receiver is only allowed to see their outputs they don't necessarily need to receive the audit info as they already have it after `GetRecipientIdentity()` is called.